modules/core/test/v2/unit/coins/trx.ts

```diff
@@ -73,20 +73,15 @@ describe('TRON:', function() {
   }));
 
   it('should sign a half signed tx', () => {
-    let tx = basecoin.signTransaction(signTxOptions);
-    let txHex = tx.halfSigned.txHex;
-    txHex = JSON.parse(txHex)
-    txHex.txID.should.equal(signTxOptions.txPrebuild.txInfo.txid);
-    txHex.raw_data.contractType.should.equal(0);
-    const contract = txHex.raw_data.contract;
-    contract.ownerAddress.should.equal(bitgoAccountLib.Trx.Utils.getBase58AddressFromHex(signTxOptions.txPrebuild.txInfo.from))
```

Comment:
why did we remove all these?

Reply:
These accessors are tested in another test. Here we care about the JSON format output and verify them with the equalities above.

`(JSON.stringify(signedTxJson.raw_data) === JSON.stringify(unsignedTxJson.raw_data)).should.be.ok` tests that every field in the transaction matches the original one, including the data type.